fastlane/Fastfile

```diff
@@ -172,40 +298,52 @@ platform :ios do
         end
     end
 
-    desc "Build for release to AppStore without symbols"
-    lane :build_for_release_without_symbols do |options|
+    # "Run security plans"
+    def security_plans(options)
+        build = Build.new(options: options)
+
         xcode_version = options[:xcode_version]
 
+        ios_version = "16.2"
         if !xcode_version.nil? 
             xcversion(version: xcode_version)
-        end
 
-        build = Build.new(options: options)
-        export_method = options[:export_method] || build.export_method
+			if xcode_version == "14.2"
+                ios_version = "16.2"
+            elsif xcode_version == "13.2.1"
+                ios_version = "15.2"
+            elsif xcode_version == "13.1"
+                 ios_version = "15.0"
+            elsif xcode_version == "12.4"
+                ios_version = "14.4"
+            end
+        end
 
-        build_app(
-            scheme: "Wire-iOS",
-            configuration: build.configuration,
-            export_method: export_method,
-            export_options: {"iCloudContainerEnvironment": "Production"},
-            derived_data_path: "DerivedData",
-            archive_path: build.archive_path(with_filename: true),
-            buildlog_path: build.build_path,
-            output_directory: build.artifact_path(with_filename: false),
-            output_name: build.filename,
-            include_bitcode: false,
-            include_symbols: false,
-            xcargs: "BUILD_NUMBER=#{build.build_number}"
-        )
+        for scheme in ["Wire-iOS", "WireDataModel", "WireSyncEngine"] do
+            run_tests(
+                testplan: "SecurityTests",
+                scheme: scheme,
+                skip_detect_devices: true,
+                configuration: "Debug",
+                # test_without_building: true,
+                sdk:"iphonesimulator",
+                destination: "platform=iOS Simulator,name=iPhone 14,OS=#{ios_version}",
```

Comment:
add note to change device once we upgrade / deprecate the iOS version.
maybe we should have an model version here in fastlane that we can change at just one place. (for Github actions and CI)

Reply:
yeap, definietely